pytomorrowio/const.py

```diff
@@ -6,8 +6,8 @@
 
 MAX_FIELDS_PER_REQUEST = 55
 
-HEADER_DAILY_API_LIMIT = "X-RateLimit-Limit-Day"
-HEADER_REMAINING_CALLS_IN_SECOND = "X-RateLimit-Remaining-Second"
+HEADER_DAILY_API_LIMIT = "X-RateLimit-Limit-day"
+HEADER_REMAINING_CALLS_IN_SECOND = "X-RateLimit-Remaining-second"
```

Comment:
HTTP headers names should be handled case-insensitively.

Reply:
I have a support ticket open about them not sending the headers, which are still documented in their docs

Reply:
After further review, since the code is using [CIMultiDict](https://multidict.readthedocs.io/en/stable/) it is already case-insensitive. Changing the case of these should be irrelevant.

Reply:
yeah - reverted it but it was a noop change. The real problem is that they aren't sending the right headers